### PR TITLE
[JW8-11174] Add Tizen ads controls

### PR DESCRIPTION
**Pull request by mamaddox**
### This PR will...
* Build on the Tizen controls created in [PR: Add Tizen Controls and Controls Loading](https://github.com/jwplayer/jwplayer/pull/3698). 
* This includes adding remote handling and styling for the controls during Ad Playback.

### Why is this Pull Request needed?
Tizen App
### Are there any Pull Requests open in other repos which need to be merged with this?
Commercial PR: https://github.com/jwplayer/jwplayer-commercial/pull/7656
Connects to Tizen Controls PR: https://github.com/jwplayer/jwplayer/pull/3698
#### Addresses Issue(s):

[JW8-11174](https://jwplayer.atlassian.net/browse/JW8-11174)

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by mamaddox** [comment]
test this please

File: src/js/view/controls/tizen/tizen-controlbar.ts [hunk 1]
```diff
@@ -62,6 +66,7 @@ export default class TizenControlbar extends Controlbar {
     activeButton: Button | null;
     el: HTMLElement;
     nextUpToolTip: NextUpToolTip;
+    adSkipButton: any;
```
Comment (jnatalzia) [inline src/js/view/controls/tizen/tizen-controlbar.ts:69]: is there any way to explicitly type this?

Comment (mamaddox) [reply src/js/view/controls/tizen/tizen-controlbar.ts:69]: i'm not sure if there is since this component exists in `jwplayer-commercial`

**Comment by robwalch** [comment]
> the changes here look good, i'd be curious to get @robwalch 's thoughts on ad logic being present in OSS in general (unsure if this is related to the import issues or if you've discussed it already but wanted to surface just in case!)

Yeah, this work could be done in commercial but not without overriding the controls loader module. In general, player UI work is public (part of this project), unless it's a component that supports a commercial feature (recs, ads, etc...).

**Comment by mamaddox** [comment]
test this please

**Comment by mamaddox** [comment]
test this please

**Pull request merged**